en/08_Changelogs/6.0.0.md

```diff
@@ -32,6 +32,8 @@ CanonicalURLMiddleware::singleton()->setEnabledEnvs([
 
 ### Other new features
 
+- Native PHP arrays can now be passed into templates and iterated over with `<% loop $MyArray %>` as though they were [`ArrayList`](api:SilverStripe\View\ViewableData) objects. You can get the count using `$Count` and use `<% if $ArrayList %>` as a shortcut for `<% if $ArrayList.Count %>`. Other functionality from `ArrayList` such as filtering and sorting cannot be used on arrays.
```

Comment:
Leaving this not done for now pending a final decision on the main PR - but if it's merged as-is (i.e. wrapping in `ArrayList`) I'll make this change.

```suggestion
- Native indexed PHP arrays can now be passed into templates and iterated over with `<% loop $MyArray %>`. Under the hood they are wrapped in [`ArrayList`](api:SilverStripe\View\ViewableData), so you can get the count using `$Count` and use `<% if $ArrayList %>` as a shortcut for `<% if $ArrayList.Count %>`. Other functionality from `ArrayList` such as filtering and sorting cannot be used on arrays since they don't have keys to filter or sort against.
    - Note that associative arrays will continue to be wrapped in `ArrayData`, which means you can access their values by key but may not be able to iterate over them.
```